tools/script_extractor.py

```diff
@@ -230,9 +230,9 @@ def createList(): # this is a func just so all this can go at the bottom
 	("Func_d1b3", "", DO_NOT_QUIT),
 	("ScriptCommand_QuitScriptFully", "", QUIT_SPECIAL),
 	("Func_d244", "q", DO_NOT_QUIT),
-	("Func_d24c", "q", DO_NOT_QUIT),
+	("ScriptCommand_ShowMultichoiceTextbox_ChooseDeckToDuelAgainst", "q", DO_NOT_QUIT),
```

Comment:
I appreciate replacing the names in here, and I can take it over from here if you'd like. However, as it stands currently the script extractor can't use these new functions because the arguments ("q") tell it to quit. for this one in particular I think it'd just be blank? I'm not really sure since there's no use in context